assets/less/content.less

```diff
@@ -13,3 +13,7 @@
   @import './content/footer';
   @import './content/bottom-actions.less';
 }
+
+body:not(.night-mode) .content-inner img[src*="#gh-dark-mode-only"] {
```

Comment:
Actually now I wonder if we can do it in `general.less` with a fancy selector like `body:not(.night-mode) & img[src*="#gh-dark-mode-only"]`, but either is fine ^^